jujugui/static/gui/src/app/components/accordion-section/_accordion-section.scss

```diff
@@ -0,0 +1,20 @@
+.accordion-section {
+  border: 1px solid $color-mid-light;
+  border-left: 0;
+  border-right: 0;
+
+  &__title {
+    padding: 1rem 1rem 1rem 0;
+    cursor: pointer;
+
+    em {
```

Comment:
For this instance I think it's ok - there's current vanilla talk around form labels and defining things as optional/ required. If a pattern comes from the talks we should implement it here, for now this is specific enough.